app/models/proposal.rb

```diff
@@ -173,6 +173,11 @@ def subscribers
     results.compact.uniq
   end
 
+  def subscribers_except_future_step_users
+    results = currently_awaiting_step_users + individual_steps.completed.map(&:user) + observers + [requester]
```

Comment:
I was under the impression that we *never* email delegates (always email mailboxes instead)...but now that I think it over more, that is more of an assumption than anything else. 

paging @phirefly for clarification -- do we want to email mailboxes when a comment is created, or if there is a delegate who already approved that proposal, do we want the comment email to go to that delegate?

current behavior is that we email mailboxes only, not delegates